Gemfile.lock

```diff
@@ -1,75 +1,75 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (6.0.2.2)
-      actionpack (= 6.0.2.2)
+    actioncable (6.0.3.2)
+      actionpack (= 6.0.3.2)
       nio4r (~> 2.0)
       websocket-driver (>= 0.6.1)
-    actionmailbox (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      activejob (= 6.0.2.2)
-      activerecord (= 6.0.2.2)
-      activestorage (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
+    actionmailbox (6.0.3.2)
+      actionpack (= 6.0.3.2)
+      activejob (= 6.0.3.2)
+      activerecord (= 6.0.3.2)
+      activestorage (= 6.0.3.2)
+      activesupport (= 6.0.3.2)
       mail (>= 2.7.1)
-    actionmailer (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      actionview (= 6.0.2.2)
-      activejob (= 6.0.2.2)
+    actionmailer (6.0.3.2)
+      actionpack (= 6.0.3.2)
+      actionview (= 6.0.3.2)
+      activejob (= 6.0.3.2)
       mail (~> 2.5, >= 2.5.4)
       rails-dom-testing (~> 2.0)
-    actionpack (6.0.2.2)
-      actionview (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
+    actionpack (6.0.3.2)
+      actionview (= 6.0.3.2)
+      activesupport (= 6.0.3.2)
       rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actiontext (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      activerecord (= 6.0.2.2)
-      activestorage (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
+    actiontext (6.0.3.2)
+      actionpack (= 6.0.3.2)
+      activerecord (= 6.0.3.2)
+      activestorage (= 6.0.3.2)
+      activesupport (= 6.0.3.2)
       nokogiri (>= 1.8.5)
-    actionview (6.0.2.2)
-      activesupport (= 6.0.2.2)
+    actionview (6.0.3.2)
+      activesupport (= 6.0.3.2)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activejob (6.0.2.2)
-      activesupport (= 6.0.2.2)
+    activejob (6.0.3.2)
+      activesupport (= 6.0.3.2)
       globalid (>= 0.3.6)
-    activemodel (6.0.2.2)
-      activesupport (= 6.0.2.2)
-    activerecord (6.0.2.2)
-      activemodel (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
-    activestorage (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      activejob (= 6.0.2.2)
-      activerecord (= 6.0.2.2)
+    activemodel (6.0.3.2)
+      activesupport (= 6.0.3.2)
+    activerecord (6.0.3.2)
+      activemodel (= 6.0.3.2)
+      activesupport (= 6.0.3.2)
+    activestorage (6.0.3.2)
+      actionpack (= 6.0.3.2)
+      activejob (= 6.0.3.2)
+      activerecord (= 6.0.3.2)
       marcel (~> 0.3.1)
-    activesupport (6.0.2.2)
+    activesupport (6.0.3.2)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.2)
+      zeitwerk (~> 2.2, >= 2.2.2)
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
-    autoprefixer-rails (9.7.5)
+    autoprefixer-rails (9.8.6.1)
       execjs
     bindex (0.8.1)
-    bootsnap (1.4.6)
+    bootsnap (1.4.7)
```

Comment:
### [bootsnap](https://github.com/Shopify/bootsnap)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.4.6 → 1.4.7

[_[change-log](https://github.com/Shopify/bootsnap/blob/master/CHANGELOG.md), [source-code](https://github.com/Shopify/bootsnap)_]





<details>
<summary>Commits</summary>

A change of **18** commits. See the full changes on [the compare page](https://github.com/Shopify/bootsnap/compare/v1.4.6...v1.4.7).

These are the first 10 commits:
- (9e3de12) [add truffleruby back to ci](https://github.com/Shopify/bootsnap/commit/9e3de12614b17d84e0953e03f51dba2f8182fca1)
- (5cf4714) [Merge pull request #296 from Shopify/truffleruby-ci](https://github.com/Shopify/bootsnap/commit/5cf4714708bf74a6746b3a6c8ee5e69631358c98)
- (ab65b31) [Fix typos](https://github.com/Shopify/bootsnap/commit/ab65b31ec2be79584e3fe788cf25f813c64794a6)
- (2530f97) [Merge pull request #298 from pgrimaud/master](https://github.com/Shopify/bootsnap/commit/2530f97eac8f8fb411dff2b5205116169eee74ab)
- (0d4bf64) [Update CHANGELOG](https://github.com/Shopify/bootsnap/commit/0d4bf643c71d7e6413c0bfc8f79a6132f0ee45aa)
- (de968f0) [Update .rubocop.yml](https://github.com/Shopify/bootsnap/commit/de968f059be2a25d7b2ef57c84066642f5d0a279)
- (aa0925e) [adding blank line after frozen comment](https://github.com/Shopify/bootsnap/commit/aa0925e54eef05a5f2419f64ff7060a9b1287df0)
- (a7818b7) [Merge pull request #304 from pdagrawal/patch-1](https://github.com/Shopify/bootsnap/commit/a7818b7a66c0939a2340c6fa678747c45dc3fc4a)
- (cfd28c3) [Ensure that temporary file is used only by one process](https://github.com/Shopify/bootsnap/commit/cfd28c385ed58e2fbc361707d8c5ad17c54302ef)
- (2cc49bb) [Update CI to test Ruby 2.7](https://github.com/Shopify/bootsnap/commit/2cc49bbbb5eeb34e77b8ab2ac0188b49123ca493)


</details>